tasks/pipeline.py

```diff
@@ -477,6 +477,17 @@ def trigger_child_pipeline(_, git_ref, project_name, variables="", follow=True):
 
     # Fill the environment variables to pass to the child pipeline.
     for v in variables.split(','):
+        # An empty string or a terminal ',' will yield an empty string which
+        # we don't need to bother with
+        if not v:
+            continue
+        if v not in os.environ:
```

Comment:
If this not to fail while the `conductor` variables are not set? If yes I'm not sure the warning message is useful, will someone read it? And is it a real warning?